### PR TITLE
feat (#547) Should handle path found in @RequestMapping.

### PR DESCRIPTION
**Pull request by iocanel**
Resolves: #547 

The idea is simple:
If the path configured for all @RestController share a common part, that part is automatically configured and exposed in the Ingress / Route.

### CODE REVIEW

**Comment by iocanel** [comment]
FYI @kameshsampath

File: frameworks/spring-boot/src/main/java/io/dekorate/spring/apt/SpringBootWebProcessor.java [hunk 1]
```diff
@@ -30,21 +37,79 @@
...
   @Override
   public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment roundEnv) {
     if  (roundEnv.processingOver()) {
       getSession().close();
       return true;
     }
+    Set<String> paths = new HashSet<>();
+
+    for (TypeElement typeElement : annotations) {
+      LOGGER.info("Found @"+typeElement.getSimpleName());
```
Comment (geoand) [inline frameworks/spring-boot/src/main/java/io/dekorate/spring/apt/SpringBootWebProcessor.java:62]: Do we really want this to be `info`?

File: readme.md [hunk 2]
```diff
@@ -841,6 +841,48 @@ Or if you are on [OpenShift](https://openshift.com):
...
 </dependency>
 ```
+
+#### Automatic configuration
+
+For Spring Boot application, dekorate will automatically detect known annotation and will align generated manifests accordingly.
+
+##### Web annotations
+
+Dekorate tunes the generated manifest based on the presennce of web annotations in the project:
```
Comment (geoand) [inline readme.md:851]:
```suggestion
Dekorate tunes the generated manifest based on the presence of web annotations in the project:
```

File: frameworks/spring-boot/src/main/java/io/dekorate/spring/apt/SpringBootWebProcessor.java (hunk 1 above)
Comment (iocanel) [reply frameworks/spring-boot/src/main/java/io/dekorate/spring/apt/SpringBootWebProcessor.java:62]: We can get rid of this.

**Pull request merged**